en/docs/develop/identity-governance-rest-api.md

```diff
@@ -3,16 +3,164 @@ template: templates/swagger.html
 ---
 
 # WSO2 Identity Server - governance connectors admin API
+     
+This is the RESTful API for managing identity governance configurations in WSO2 Identity Server. These configurations
+ are grouped in to main 5 categories. Each category will have sub categories which are called connectors, and 
+ properties of connectors as mentioned below. <br>
 
-??? Note "Note: Click to view"
+??? "Password Policies"
```

Comment:
??? note "Password Policies"

Reply:
Without adding the word "note" it is rendered as a collapsible. So not added the word.